src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AFInitializer.java

```diff
@@ -0,0 +1,167 @@
+package org.broadinstitute.hellbender.tools.exome.allelefraction;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.apache.commons.math3.exception.MaxCountExceededException;
+import org.broadinstitute.hellbender.tools.exome.AllelicCount;
+
+import java.util.ArrayList;
+import java.util.EnumMap;
+import java.util.List;
+
+import static org.apache.commons.math3.special.Beta.*;
+
+/**
+ * Created by davidben on 11/18/15.
+ */
+public class AFInitializer {
+    public static final double INITIAL_OUTLIER_PROBABILITY = 0.01;
+    private static final double BIAS_MEAN_INITIAL = 1.0;
+    private static final double BIAS_RATE_INITIAL = 10.0;   // this is an underestimate, but starting high makes it slow for
+                                                            // mean bias to escape a bad initial guess
+    private static final int NUMBER_OF_EM_ITERATIONS = 10;
+
+    private final AFData data;
+    private AFState state;
+    private final List<Responsibility> responsibilities = new ArrayList<>();
+
+    // posterior expectations on alt minor / ref minor / outlier indicators for a het in the Allele Fraction Model
+    @SuppressWarnings("serial")
+    @VisibleForTesting
+    static class Responsibility extends EnumMap<AFIndicator, Double> {
+        // initialize naively based on assumption of no allelic bias and unknown minor fraction f
+        // that is, likelihood of alt minor is int_{0 to 1/2} f^a (1-f)^r df
+        //          likelihood of ref minor is int_{0 to 1/2} f^r (1-f)^a df
+        // these are proportional to I(1/2, a + 1, r + 1) and I(1/2, r + 1, a + 1),
+        // respectively, where I is the (incomplete) regularized Beta function
+        // note that by virtue of being regularized, these likelihoods sum to 1
+        public Responsibility(final AllelicCount count) {
```

Comment:
You should use the javadoc comment style here since this is public... ``/** .... */``